docs/components/Combobox/Combobox.stories.mdx

```diff
@@ -47,15 +47,55 @@ handling user interactions, and managing search functionality.
 Actions are an optional way to allow users to perform custom tasks such as
 adding a new item to the list of options.
 
-#### Callback
+### Selection behavior
 
-`onSelect` is the recommended callback handler that should be used in most cases. This will immediately inform
-the consumer of changes to selections as they happen.
+#### Single-select
 
-If however immediate changes are not desired, and instead it is preferable to wait until all selections or
-modifications to selections have been made before informing the consumer, `onClose` may be used. The arguments provided to this callback are
-Identical to those of `onSelect`, only the timing differs. Please note this will only
-have a noticeable effect with `multiSelect = true` since the Combobox closes immediately upon selection with single-select.
+Single-select is ComboBox's default selection mode. When a selection is made,
+the Combobox will close.
+
+#### Multi-select
+
+Add the `multiSelect` prop to allow multiple options to be selected and/or
+un-selected. In multi-select mode, the Combobox will only close when a user
+takes an explicit action to close it such as clicking outside the Combobox or
+pressing the ESC key.
+
+When `multiSelect` is enabled, a Combobox header will be displayed below the
+search input.
```

Comment:
Not sure what this combobox header is?

Reply:
@cleo-e-jobber ComboboxHeader is the section under the search that says `Select` and `Select all`.  Those labels will change based on the state of the filter/selections made, etc. Would it be more helpful to have a picture of it on the docs page?

![Screenshot 2023-10-19 at 1 49 13 PM](https://github.com/GetJobber/atlantis/assets/104797704/eb32f687-915c-4662-89ff-7f7ca94fa1ae)

Reply:
All good, now that I see the actual header, what you wrote made sense!

Reply:
https://jobber.atlassian.net/browse/JOB-80566 <- related ticket